### ISSUE TITLE
luarocks install error Ubuntu 14.04  with utility.lua

### ISSUE THREAD

**Issue by rmcdermith** [jmcnamara/xlsxwriter.lua#13]
luarock error when inflating xlsxwriter-0.0.6-1.rockspec, "can not stat 'xlsxwriter/utility.lua ': No such file or directory.

terminal line = sudo luarocks install xlsxwriter

build is aborted.  I can install anything else including the dependencies just not this. Any idea's why?

**Comment by jmcnamara** [comment]
I don't why that is. Perhaps there is a spec issue. 

@hishamhm could you check it for me, I'm temporarily away from any machines I could run it on.

John.

**Comment by moteus** [comment]
Is file name has trail space?
Rockspec in repo and in luarocks.org has no this space.
May be you use some luarocks mirror that contain wrong version of rockspec.
Try
`luarocks install https://raw.githubusercontent.com/jmcnamara/xlsxwriter.lua/master/xlsxwriter-0.0.6-1.rockspec`

**Comment by zunware** [comment]
I am having a very similar problem. 

Error message:
    Using https://raw.githubusercontent.com/rocks-moonscript-org/moonrocks-mirror/master/xlsxwriter- 0.0.6-1.src.rock... switching to 'build' mode
    cp: xlsxwriter/core.lua: No such file or directory
    
    Error: Build error: Failed installing xlsxwriter/core.lua in .../luarocks/rocks/xlsxwriter/0.0.6-1/lua/xlsxwriter/core.lua

Installed using @moteus solution works for me.

>Try
luarocks install https://raw.githubusercontent.com/jmcnamara/xlsxwriter.lua/master/xlsxwriter-0.0.6-1.rockspec

**Comment by moteus** [comment]
Just faced with same error. Problem is `xlsxwriter-0.0.6-1.src.rock` has not source. only rockspec.
So pleas reapload source on server.

**Comment by jmcnamara** [comment]
@moteus You are right. The src spec doesn't have any source. That is odd since I built it in the same way as the previous rocks which did have the source.

I'll try fix it and upload a new version.
